src/main/java/com/stablekernel/standardlib/BlockingProgressFragment.java

```diff
@@ -28,6 +28,7 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         Dialog dialog = super.onCreateDialog(savedInstanceState);
         dialog.getWindow().requestFeature(Window.FEATURE_NO_TITLE);
         dialog.getWindow().setBackgroundDrawable(new ColorDrawable(android.graphics.Color.TRANSPARENT));
+        setCancelable(false);
```

Comment:
remember, this isn't only being used on one project.  you might be right, but just remember that.  

what if it defaulted to false but you could enable it?